content/tutorials/realtime-attendance-system-iot.textile

```diff
@@ -0,0 +1,262 @@
+    }
+    buffer[len*2] = '\0';
+}
+```
+
+Every access card can be identified by a unique identifier, called UID. As you can see, in the above code, I've just included a simple UID check with a specific UID. In practice, you can have a switch case to match the UIDs with the employee names.
+
+Go ahead and verify that the sketch has no errors by clicking on the 'tick' button in the Arduino IDE. Assuming everything is good, connect your board to your computer and upload the sketch to it by clicking on the 'right arrow' button. Open the serial monitor (search icon button on the top right) to check the logs. 
+
+If you have a connection error for the MFRC522 component, it's most likely due to some loose connection. The best solution would be to solder the jump wires to the board and the RFID component.
+
+Now, we can very easily verify if the message was successfully published to Ably using the dev console in the Ably dashboard. Open it up, subscribe to the same channel to which you are publishing the messages, in this case "standup" and you should see the logs coming through whenever you present the access card in front of the RFID reader.
+
+<a href="/images/tutorials/iot-attendance/ably-dev-console.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/ably-dev-console.png" style="width: 100%" alt="Ably dev console">
+</a>
+
+You'll notice from the logs in the dev console that the data payload of the message is not infact the same string message you published from your Arduino IDE. This is due to MQTT being a binary protocol, the messages published are in fact encoded and need to be "decoded at the recipient client side":/mqtt#decoding in order to get the actual message contents out. 
+
+For the Reactor integrations, Ably does this automatically so the contents can be directly sent over the integrating application, in this case, Zapier.
+
+h2(#step4-ably-zapier). Step 4 - Triggering a Zapier endpoint from Ably via WebHooks
+
+In this step, we'll see how to create a Zapier endpoint and set up a Reactor rule in Ably to trigger the Zap whenever a message is published from the IoT device.
+
+Log into you Zapier account and click on "Make a zap". The trigger app needs to be "WebHooks by Zapier".
+
+<a href="/images/tutorials/iot-attendance/zap-setup.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/zap-setup.png" style="width: 100%" alt="Zap set up">
+</a>
+
+The action needs to be "Catch a Raw Hook", so choose that and continue. In the next screen Zapier will give you a URL, which is the WebHooks endpoint needed by Ably, so that it can trigger that URL when a message is published on a channel. Copy that out.
+
+<a href="/images/tutorials/iot-attendance/zap-wh-endpoint.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/zap-wh-endpoint.png" style="width: 100%" alt="Zap WebHooks endpoint">
+</a>
+
+In another tab, log into your Ably account, go to the Apps Dashboard. In the "Reactor" tab, click:
```

Comment:
We'll have a partial for this if you want it once the IFTTT branch is merged

Reply:
Cool, will update when it's available 👍🏼